ui/app/models/auth-config/ldap.js

```diff
@@ -1,50 +1,11 @@
 import { computed } from '@ember/object';
-import DS from 'ember-data';
 
 import AuthConfig from '../auth-config';
 import fieldToAttrs from 'vault/utils/field-to-attrs';
 import { combineFieldGroups } from 'vault/utils/openapi-to-attrs';
 
-const { attr } = DS;
-
 export default AuthConfig.extend({
   useOpenAPI: true,
-  binddn: attr('string', {
```

Comment:
notice here that when we set `useOpenAPI` to true, we can get rid of the duplicate attrs. OpenAPI gives us the attrs and sets them on the model here: https://github.com/hashicorp/vault/pull/6702/files#diff-cd3b672bded0741ca8ec360a4deb124fR37